hugo/layouts/partials/teaser--blog.html

```diff
@@ -0,0 +1,29 @@
+<article class="teaser teaser--blog">
+    {{ if .Params.Image }}
+        <div class="media">
```

Comment:
is i tcorrect we have the image class as teaser__image but the media class just as media? Also do we need both a div classs media and figure class teaser__image. ?